bin/install.sh

```diff
@@ -65,7 +74,8 @@ sudo apt-get update
 sudo apt-get purge -y python-setuptools python-pip python-pyasn1
 sudo apt-get install -y python-dev git-core libffi-dev libssl-dev
 curl -s https://bootstrap.pypa.io/get-pip.py | sudo python
-sudo pip install ansible==2.1.0.0
+
+sudo pip install ansible==2.5.3
```

Comment:
This change might break things.

Reply:
hmm, I just copied this from the experimental branch. there it does not break compatibility

Reply:
it seems you have already updated ansible in the requirements file
https://github.com/Screenly/screenly-ose/blob/master/requirements.txt#L3